docs/streams/stream-basics/message-queues/set-message-queue.md

```diff
@@ -0,0 +1,41 @@
+---
+sidebar_position: 10
+title: Set Message Queue
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+This page shows you several ways to use a stream as a message queue.
+
+To use a stream as a message queue:
+
+1. Establish a [shared subscription](../subscriptions#shared) and use the same subscription name as the other consumers. 
+1. Set the **receiver queue** for the consumers.
+
+When you subscribe to a stream, use the following configuration settings depending on your SDK:
+
+<Tabs groupId="operating-systems">
+<TabItem value="py" label="Python SDK">
```

Comment:
Make sure the order of these tabs is consistent throughout. I am pretty sure that these are reversed from the other PR I just edited.